src/app/containers/FrontPageMain/index.jsx

```diff
@@ -65,6 +66,9 @@ const FrontPageMain = ({ frontPageData }) => {
     </span>
   );
 
+  const hasUsefulLinks =
+    groups.findIndex(group => group.type === 'useful links') > -1;
```

Comment:
Should this not be `useful-links` with a hyphen like in Ares data?

Reply:
yep